### PR TITLE
Remove the site_title from the docs

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Removed the site_title attr from the docs

## QA

- Check all site_titles have been removed.

### CODE REVIEW

File: docs/en/metadata.yaml [hunk 1]
```diff
@@ -119,8 +119,8 @@ navigation:
     - location: utilities/vertically-center.md
       title: Vertically center
 
-    - location: utilities/visibility.md
-      title: Visibility
+    - location: utilities/show.md
```
Comment (nottrobin) [inline docs/en/metadata.yaml]: Why is this change here?

Comment (nottrobin) [inline docs/en/metadata.yaml]: Sorry, I discovered. But shouldn't you add "hide.md" too?

Comment (anthonydillon) [inline docs/en/metadata.yaml]: Sorry, should have mentioned. It a drive-by fix. The pattern name was changed but the navigation wasn't.

**Comment by anthonydillon** [comment]
Done.
